src/libraries/System.Private.CoreLib/src/System/IntPtr.cs

```diff
@@ -17,8 +18,9 @@
 namespace System
 {
     [Serializable]
+    [StructLayout(LayoutKind.Sequential)]
```

Comment:
Why is this needed? structs are Sequential by default.

Reply:
We also have it on the other primitive types, such as `Int32`. It was added for consistency.

If we don't add it here, we should likely have a separate PR that removes it elsewhere.